x-pack/plugins/canvas/canvas_plugin_src/uis/arguments/date_format/index.tsx

```diff
@@ -16,9 +16,13 @@ import { SetupInitializer } from '../../../plugin';
 
 const { DateFormat: strings } = ArgumentStrings;
 
+const getDateFormatArgInput = (defaultDateFormats: ComponentProps['dateFormats']) => (
+  props: Omit<ComponentProps, 'dateFormats'> & Partial<Pick<ComponentProps, 'dateFormats'>>
+) => <DateFormatArgInput dateFormats={defaultDateFormats} {...props} />;
+
 export const dateFormatInitializer: SetupInitializer<ArgumentFactory<ComponentProps>> = (
   core,
-  plugins
+  _
```

Comment:
just interesting why it was replaced to _

Reply:
If we are not using it, and not planning to use at the current initializer, why should it be name? I like such a style)

Reply:
it confuse a little, is it possible to remove that argument at all?

Reply:
![image](https://user-images.githubusercontent.com/20072247/128184810-645c875b-f1c7-43ca-9892-cc82f2632540.png)

no issues

Reply:
Sure, but each initializer has its own uniformed signature. If somebody would like to use that `plugins.canvas` inside the initializer, it would be very difficult to recognize, that we have that param.

Reply:
it's not, all what you need in `SetupInitializer<ArgumetFactory<ComponentProps>>` type. 

![image](https://user-images.githubusercontent.com/20072247/128247756-690547fd-30f2-4c74-aab8-9f6d1ae77ef6.png)

Plus if in future someone needs that param it should be renamed back =( I recommend you to remove it or rename it back

Reply:
Done.